src/python/pants/backend/python/register.py

```diff
@@ -84,6 +85,8 @@ def rules():
         *setuptools.rules(),
         *ipython.rules(),
         *pytest.rules(),
+        # Macros
+        *pants_requirements.rules(),
```

Comment:
Question that I don't know the answer to: should this be in a separate backend? The implication would be that the first step of plugin development would be "enable to the `plugin_development` backend" or something. But it might avoid some confusion of having it listed in the `python` backend as a whole.

Reply:
Yeah, I like that. We have a tonnnn of targets right now with `./pants help`.

But hey, that makes me wonder, why even have `pantsbuild.pants.testutil` anymore?? I theoretically get the argument that `pantsbuild.pants` should be as small as possible for end-users. But in reality, `pantsbuild.pants.testutil` is only 23.2kb. I'll bring this to Slack

Reply:
Answered in Slack. It's useful to have `pantsbuld.pants.testutil` (or the extra `pantsbuild.pants[testutil]`) so that our test support can depend on extra third-party requirements. At that point, we can stick with the two dists because it's roughly a different typing of the single dist with an extra.

So, `pants_requirements` still makes sense. We only need to create a dedicated backend.